### PR TITLE
feat(EMS-2300): No PDF - Your buyer - start page

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR adds a new start page to the "Your buyer" section of an application.

## Resolution :heavy_check_mark:
- Create new cypress command to start the "Your buyer" section, `cy.startInsuranceYourBuyerSection`. Update various E2E commands/functions.
- Add E2E test for the new start page.
- Create new UI route, controller, content strings.
- Update the "your buyer" task list item to point to the new start page.
- Update the "section start" nunjucks template to conditionally render a list outro.
- Update "Your business - check your answers" to redirect to the new "Your buyer" start page, instead of the "Company or organisation" form.

## Miscellaneous :heavy_plus_sign:
- Various minor destructuring improvements.
- Fixed some typos.
- Create a new `assertSectionStartContent` cypress object to assert various pieces of "section start" page content.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.buyer;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Your buyer - start page - As an exporter, I want to provide the details on the buyer of my export trade, So that UKEF can gain clarity on the buyer history as part of due diligence', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js:30]: Capital S for start page? not a blocker

Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js:30]: Great spot - I have a small tech improvement for "section start" controllers after this, will address there.

File: e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    it('renders a list outro', () => {
+      cy.checkText(listOutro(), CONTENT_STRINGS.LIST.OUTRO);
+    });
+
+    it('renders an outro', () => {
+      cy.checkText(outro(), CONTENT_STRINGS.OUTRO);
+    });
+
+    it('renders a `start now` link', () => {
+      cy.checkLink(
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js:91]: could we make these a function so can be used for all start pages

**Review by Zainzzkk — APPROVED** [review]

**Review by Zainzzkk — APPROVED** [review]

File: e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js]: Created a new assertion object for this, `assertSectionStartContent`. Will apply to all other start pages in the next PR.

**Pull request merged**